src/localization_plugin.cpp

```diff
@@ -98,6 +98,10 @@ void LocalizationPlugin::configure(tue::Configuration config)
 
     config.value("num_particles", num_particles_);
 
+    double tmp_transform_tolerance = 0.1;
```

Comment:
magic number, any way to make this clearer? Perhaps make it configurable?

Reply:
This is the default. Is change in the next line by the config reader.